apps/mobile/src/components/DrawerMenu/DrawerMenu.js

```diff
@@ -8,9 +8,10 @@ import ContextSwitchMenu from 'components/ContextSwitchMenu/ContextSwitchMenu'
 
 export default function DrawerMenu (props) {
   const insets = useSafeAreaInsets()
-  const { myHome, groupSlug } = useRouteParams()
+  const { groupSlug } = useRouteParams()
+  const [{ fetching }] = useCurrentGroup({ setToGroupSlug: groupSlug })
 
-  useCurrentGroup({ setToGroupSlug: groupSlug })
+  if (fetching) return null
```

Comment:
Won't this hide the ContextSwitchMenu while you select a group and it is fetching the full group details?

Reply:
Indeed it will, good catch. Fixed.